src/__stories__/image-story.tsx

```diff
@@ -1,72 +1,96 @@
 import * as React from 'react';
-import {Image, Stack, Text3, Title1, skinVars} from '..';
-import {isRunningAcceptanceTest} from '../utils/platform';
+import {Image} from '..';
 import usingVrImg from './images/using-vr.jpg';
 
+import type {AspectRatio} from '../image';
+
+const typeOptions = ['width and height', 'width and aspect ratio', 'full width'];
+
 export default {
     title: 'Components/Primitives/Image',
+    argTypes: {
+        type: {
+            options: typeOptions,
+            control: {type: 'select'},
+        },
+        width: {
+            control: {type: 'range', min: 200, max: 800, step: 10},
+            if: {arg: 'type', neq: 'full width'},
+        },
+        height: {
+            control: {type: 'range', min: 200, max: 800, step: 10},
+            if: {arg: 'type', eq: 'width and height'},
+        },
+        aspectRatio: {
+            options: ['1 1', '16 9', '7 10', '4 3', '0'],
```

Comment:
why space instead of `:` ?
```suggestion
            options: ['1:1', '16:9', '7:10', '4:3', '0'],
```